docs/index.md

```diff
@@ -226,3 +226,39 @@ The Snowflake provider will use the following order of precedence when determini
+- [snowflake_masking_policy_grant](./resources/masking_policy_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_materialized_view_grant](./resources/materialized_view_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_pipe_grant](./resources/pipe_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_procedure_grant](./resources/procedure_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_resource_monitor_grant](./resources/resource_monitor_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_role_grants](./resources/role_grants) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_row_access_policy_grant](./resources/row_access_policy_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_schema_grant](./resources/schema_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_sequence_grant](./resources/sequence_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_stage_grant](./resources/stage_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_stream_grant](./resources/stream_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_table_grant](./resources/table_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_tag_grant](./resources/tag_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_task_grant](./resources/task_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_unsafe_execute](./resources/unsafe_execute)
+- [snowflake_user_grant](./resources/user_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_view_grant](./resources/view_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+- [snowflake_warehouse_grant](./resources/warehouse_grant) - use [snowflake_grant_privileges_to_account_role](./resources/grant_privileges_to_account_role) instead
+
+
+## Currently deprecated datasources
+
+- [snowflake_role](./resources/role) - use [snowflake_roles](./resources/roles) instead
```

Comment:
Is ./resources correct or it should be ./data-sources?

Reply:
hmmm, it's probably wrong in this case, I will check

Reply:
Fixed